sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -97,7 +93,8 @@
  * {@link ServiceBusAdministrationClientBuilder}.
  *
  * <p><strong>Create a queue</strong></p>
- * <!-- src_embed com.azure.messaging.servicebus.administration.servicebusadministrationasyncclient.createqueue#string -->
+ * <!-- src_embed com.azure.messaging.servicebus.administration.servicebusadministrationasyncclient.createqueue#string
+ * -->
```

Comment:
* Does this break the snippet?
* Can we scope the changes to just code changes? It makes it harder to review having to look at whitespace changes from auto-formatting.